Client.php

```diff
@@ -129,13 +128,6 @@ protected function _run(HoaSocket\Node $node)
 
                     break;
 
-                case 433: // ERR_NICKNAMEINUSE
-                    throw new Exception(
-                        'Nickname %s is already in use.',
-                        0,
-                        $node->getUsername()
```

Comment:
The exception will be caught by the global `try`/`catch`, and it will be fired as an `error` event. In the listener, it will be able to get the current node, so it's fine.

Thoughts?